DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -1090,12 +1091,25 @@ def onCancel(self):
 
 class GoogleCloudPlatform(object):
 
+  def findgcloud(self):
+    gcloudPath = shutil.which('gcloud')
+    if gcloudPath:
+      return gcloudPath
+
+    if sys.platform in ("linux", "darwin"):
+      # Try a second way to find gcloud path
+        shell = os.environ.get("SHELL")
+        if shell:
+          process = slicer.util.launchConsoleProcess([shell, "-c", "which gcloud"])
+          process.wait()
+          gcloudPath = process.stdout.read().strip()
+          if os.path.exists(gcloudPath):
+            return gcloudPath
+
+    logging.error(f"Unable to locate gcloud, please install the Google Cloud SDK")
+
   def gcloud(self, subcommand):
-   
-    import shutil 
-    args = [shutil.which('gcloud')]
-    if (None in args):
-      logging.error(f"Unable to locate gcloud, please install the Google Cloud SDK")
+    args = [self.findgcloud()]
```

Comment:
It's probably trivial amount of overhead but maybe we should do this once and cache the result.

Reply:
I cached it in a class variable. Probably overkill since the auth token received from `gcloud` gets cached in the `DICOMwebClient` object and there are only three other places the `gcloud` executable is used (in an optional way to select the dicomweb url)